wallet/chain/x/backend.go

```diff
@@ -4,8 +4,6 @@
 package x
 
 import (
-	"fmt"
-
 	stdcontext "context"
```

Comment:
@StephenButtolph Why is this aliased in wallet files?

Reply:
We have a `context` already: https://github.com/ava-labs/avalanchego/blob/master/wallet/chain/x/context.go#L24

Reply:
ty